### PR TITLE
Show output formatting

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

**Comment by burnout87** [comment]
Output of a Dummy request for `isgri_lc` is:

`ID=0 prod_name=isgri_lc_0_OAO1657m415  meta_data: {'src_name': 'OAO 1657-415', 'time_bin': 0.0115739687598762, 'time': 'TIME', 'rate': 'RATE', 'rate_err': 'ERROR'}`

File: oda_api/api.py
Comment (volodymyrss) [inline oda_api/api.py:934]: This makes the data available as attributes of the object.
One may argue how convenient it is, surely it is sometimes and sometimes it causes issues.

But, as of now, it **must remain** since it is interface used in various notebooks, some of them published.

File: tests/test_basic.py
Comment (volodymyrss) [inline tests/test_basic.py:43]: Can we also here check that the result is available as `products.isgri_lc_0_OAO1657m415` and also in products._n_list and products._p_list

**Review by volodymyrss — CHANGES_REQUESTED** [review]

**Review by volodymyrss — APPROVED** [review]
For me it's ok. I do not know if that's what @ferrigno wanted, I did not look into the issue too closely.

**Comment by burnout87** [comment]
> For me it's ok. I do not know if that's what @ferrigno wanted, I did not look into the issue too closely.

Waiting for his approval indeed

**Comment by volodymyrss** [comment]
> > For me it's ok. I do not know if that's what @ferrigno wanted, I did not look into the issue too closely.
> 
> Waiting for his approval indeed

For @ferrigno it could be useful to look into the [test](https://github.com/oda-hub/oda_api/blob/show-output-formatting/tests/test_basic.py#L43) to understand if that's what you expected to happen.

**Comment by burnout87** [comment]
> For me it's ok. I do not know if that's what @ferrigno wanted, I did not look into the issue too closely.

Ah BTW, about my comment here:

[https://github.com/oda-hub/oda_api/issues/44#issuecomment-863867220](https://github.com/oda-hub/oda_api/issues/44#issuecomment-863867220)

Could this be an issue on the JEM-X side?

**Comment by volodymyrss** [comment]
> > For me it's ok. I do not know if that's what @ferrigno wanted, I did not look into the issue too closely.
> 
> Ah BTW, about my comment here:
> 
> [#44 (comment)](https://github.com/oda-hub/oda_api/issues/44#issuecomment-863867220)
> 
> Could this be an issue on the JEM-X side?

Well that's a property, do not know if it is an issue, the way the source names are extracted they are guaranteed to squash some characters:

https://github.com/oda-hub/dispatcher-plugin-integral/blob/master/cdci_osa_plugin/osa_lightcurve_query.py#L174

It's confusing the variable is called source name. Actually source name is stored in the file and can be extracted if necessary.
The concern here in this issue though is just mismatch between printing and assigning.

Anyway these are not big issues.

**Comment by ferrigno** [comment]
Source name should be derived as for ISGRI 
https://github.com/oda-hub/dispatcher-plugin-integral/blob/47c362111dfdee32c9d46fc5d15ae666af4ab0cb/cdci_osa_plugin/osa_lightcurve_query.py#L142

The way in which it is derived now for JEM-X
https://github.com/oda-hub/dispatcher-plugin-integral/blob/master/cdci_osa_plugin/osa_lightcurve_query.py#L174
leads to use a space instead of Minus or plus

So it is not exactly what I wanted for JEM-X.

**Comment by ferrigno** [comment]
If you implement the change, do not transfer the "TIMEDEL" stuff to JEM-X, it will not work.

**Comment by volodymyrss** [comment]
Indeed:

> Well that's a property, do not know if it is an issue, the way the source names are extracted they are guaranteed to squash some characters:
> 
> https://github.com/oda-hub/dispatcher-plugin-integral/blob/master/cdci_osa_plugin/osa_lightcurve_query.py#L174
> 
> It's confusing the variable is called source name. Actually source name is stored in the file and can be extracted if necessary.
> The concern here in this issue though is just mismatch between printing and assigning.
> 
> Anyway these are not big issues.

But it is a different issue, and belongs to dispatcher plugin.

This PR one here presumably had something to do with that what is printed is not what is available. Is this not what you explained?

The fact that what is provided by jemx is not unmodified src names can not be addressed by oda_api PR.

**Comment by burnout87** [comment]
> Indeed:
> 
> > Well that's a property, do not know if it is an issue, the way the source names are extracted they are guaranteed to squash some characters:
> > https://github.com/oda-hub/dispatcher-plugin-integral/blob/master/cdci_osa_plugin/osa_lightcurve_query.py#L174
> > It's confusing the variable is called source name. Actually source name is stored in the file and can be extracted if necessary.
> > The concern here in this issue though is just mismatch between printing and assigning.
> > Anyway these are not big issues.
> 
> But it is a different issue, and belongs to dispatcher plugin.
> 
> This PR one here presumably had something to do with that what is printed is not what is available. Is this not what you explained?
> 
> The fact that what is provided by jemx is not unmodified src names can not be addressed by oda_api PR.

That was indeed my point. If we want to change the behavior, specifically for JEM-X then we'll do so on  a dedicated PR

**Review by ferrigno — APPROVED** [review]

**Comment by ferrigno** [comment]
Right, I approve this one and close it

**Comment by ferrigno** [comment]
I should have not closed it, I let you merge and close.

**Pull request merged**
